Pycloud Deployer/config.py

```diff
@@ -0,0 +1,30 @@
+import os
+import logging
+from dotenv import load_dotenv
+
+logger =  logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('%(levelname)s - %(asctime)s - %(message)s')
+
+# Configure console log output with desired format (formatter)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(formatter)
+logger.addHandler(console_handler)
+
+# Configure file log output to app.log with desired format
+file_handler = logging.FileHandler('app.log')
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
+logger.info("Loading environment variables from .env file")
+try:
+    # Load variables in .env file into ENVIRONMENT
+    load_dotenv(".env")
+    logger.info(".env file loaded successfully.")
+except FileNotFoundError:
+    logger.critical("The file '.env' could not be found.\n Terminating script...")
+    SystemExit(1)
```

Comment:
I could be wrong but I think you should raise SystemExit(1) to properly exit the script instead of  just calling SystemExit(1) .
Verify first before making a change